zulipterminal/ui_tools/views.py

```diff
@@ -587,5 +588,108 @@ def __init__(self, controller: Any) -> None:
 
     def keypress(self, size: Tuple[int, int], key: str) -> str:
         if is_command_key('GO_BACK', key) or is_command_key('HELP', key):
-            self.controller.exit_help()
+            self.controller.exit_pop_up()
         return super(HelpView, self).keypress(size, key)
+
+
+class MsgInfoView(urwid.ListBox):
+    def __init__(self, controller: Any, msg: Any) -> None:
+        self.controller = controller
+        self.msg = msg
+
+        if msg['type'] == 'stream':
+            recipient = msg['display_recipient'] + " -> " + msg['subject']
+        elif not msg['display_recipient'][0]['full_name'] ==\
+                msg['sender_full_name']:
+            recipient = msg['display_recipient'][0]['full_name']
+        else:
+            recipient = 'Self'
+
+        msg_info = {
```

Comment:
Using a dict here doesn't ensure that the results will always be in this order, at least for early enough versions of python3. For 3.6+ we should be good, but for now an `OrderedDict` may be preferable - we use this elsewhere for this reason.